src/Interpolations.jl

```diff
@@ -4,6 +4,7 @@ export
     interpolate,
     interpolate!,
     extrapolate,
+    scale,
```

Comment:
I don't think there's need for that; there's nothing you can do with that constructor that `scale` can't do (unlike `FilledExtrapolation`, where there's an actual use case for explicitly using the constructor).

We should, though, probably export `AbstractInterpolation`, to make it easier to create functions that use these objects. But the whole point of the composability design is that you shouldn't have to know which the concrete type is.